submissions/antonkottans/memory-pair-game/script.js

```diff
@@ -0,0 +1,250 @@
+const pipe = (...arrayOfFunctions) => {
+    if (arrayOfFunctions.length === 0) return undefined;
+    else if (arrayOfFunctions.length === 1) return arrayOfFunctions[0]();
+    return arrayOfFunctions.reduce((prev, func) => {
+        return func(prev);
+    }, undefined);
+};
+let timerId = null;
+
+const state = {
+    timerId: null,
+    guessedPairsAmount: 0,
+    flipped: false,
```

Comment:
This value is defined as `boolean` here, but later in the code objects are placed here. If you want some placeholder that will mean "nothing is here yet", use `null`.